bittorrent/event_test.go

```diff
@@ -22,8 +23,10 @@ func TestNew(t *testing.T) {
 	}
 
 	for _, tt := range table {
-		got, err := NewEvent(tt.data)
-		require.Equal(t, err, tt.expectedErr, "errors should equal the expected value")
-		require.Equal(t, got, tt.expected, "events should equal the expected value")
+		t.Run(fmt.Sprintf("%#v expecting %s", tt.data, tt.expectedErr), func(t *testing.T) {
```

Comment:
The expected `err` is nil most of the time, looks pretty weird in the travis log, see [here](https://travis-ci.org/chihaya/chihaya/jobs/323034858#L555)